internal/app/service.go

```diff
@@ -430,6 +430,22 @@ func (svc *Service) AddFunctionsPipelineForTopics(id string, topics []string, tr
 	return nil
 }
 
+// SetFunctionsPipelineTransforms updates the functions pipeline for the specified id
+func (svc *Service) SetFunctionsPipelineTransforms(id string, transforms ...interfaces.AppFunction) error {
```

Comment:
This is not needed. Should use `SetDefaultFunctionsPipeline` or `AddFunctionsPipelineForTopics`

Reply:
How do I update the transforms of a specific pipeline when my configuration changed? Call `RemoveAllFunctionPipelines` firstly and `SetDefaultFunctionsPipeline` or `AddFunctionsPipelineForTopics` again?